### PR TITLE
Improve Bottom Navigation

### LINKED ISSUE #242: Improve Bottom Navigation UI
## Is your feature request related to a problem? Please describe.

The current bottom navigation doesn't look nicer. It can be improved by giving some border radius and shadows.

## Proposed Design

Screenshot 2021-04-10 000930

Any suggestions @Abhishek-kumar09

### PR DESCRIPTION
**Pull request by faisaljamil25**
**Issue Number**

fixes #242 

**Describe the changes you've made**

- Changed nav icons
- Changed font color to secondary color of the theme
- Changed background shadow
- Removed bottom border for active nav
- Added border radius

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

`NA`

**Before**

Screenshot 2021-04-10 142657

**After**

With Border radius (added in the PR)
Screenshot 2021-04-10 141907

Without Border radius
Screenshot 2021-04-10 141824

**Checklist**

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: src/layouts/MainLayout/BottomNav.css [hunk 1]
```diff
@@ -1,9 +1,13 @@
-.MuiBottomNavigationAction-root.Mui-selected {
+/* .MuiBottomNavigationAction-root.Mui-selected {
```
Comment (sakshi-choudhary) [inline src/layouts/MainLayout/BottomNav.css:1]: Hey, @faisaljamil25 I think you should not keep any commented codes.

Comment (faisaljamil25) [reply src/layouts/MainLayout/BottomNav.css:1]: Hey, @sakshi-choudhary actually I was waiting for mentors' confirmation whether I should remove the bottom border or not. That's why I commented for now. I will either remove it if I get confirmation or revert back.

Comment (sakshi-choudhary) [reply src/layouts/MainLayout/BottomNav.css:1]: okay

**Review by Abhishek-kumar09 — COMMENTED** [review]
Red color is not looking professional for the app which has bot blue and red as their theme color.

Also the rounded border needs to be reviewed other than me to be merged in the final code.

**Pull request closed without merging**